rosette/rbl/rosette/configuration.rbl

```diff
@@ -44,5 +44,4 @@
 (load 'apropos		'silent)
 (load 'space 'silent)
 (load 'primitives 'silent)
-
-(load 'tests/README 'silent)
+(exit)
```

Comment:
I think this would cause the program to exit immediately after executing. In other words, you couldn't use the REPL.

Reply:
Yes.  Remove the "(exit)".